bin/varnishtest/tests/u00015.vtc

```diff
@@ -1,5 +1,7 @@
 varnishtest "varnishstat -j"
 
+feature ignore_unknown_macro
```

Comment:
I would rather mark the test as skipped instead of faking its results:

```
feature cmd "${PYTHON} -c 'import json'"
```

Reply:
ok, so split the test in two, `varnishstat -j` on one side, and `varnishstat -j | ${PYTHON} ...` on the other?

Reply:
No, with my suggested `feature` statement the test is skipped if you don't have the `json` module!

Reply:
Alternatively, can we maybe use `jq` for this?

Reply:
I think this is overall cleaner:

```vcl
varnishtest "varnishstat -j"

feature cmd "command -v jq"

server s1 -repeat 5 {
        rxreq
        txresp
} -start

varnish v1 -vcl+backend {} -start

client c1 -repeat 5 {
        txreq
        rxresp
} -run

varnish v1 -expect MAIN.client_req == 5

shell -expect 1 {
        varnishstat -n ${v1_name} -j |
        jq .version
}

shell -expect 5 {
        varnishstat -n ${v1_name} -j |
        jq '.counters."MAIN.client_req".value'
}

shell -expect {"Good client requests received"} {
        varnishstat -n ${v1_name} -j |
        jq '.counters."MAIN.client_req".description'
}
```

Reply:
> No, with my suggested feature statement the test is skipped if you don't have the json module!

if you have `python3` then you should have the `json` module, so it's really just a matter of testing for python. But I feel my intent was misunderstood: I still want to make sure that `varnishstat -j` works even if I can't test the output, hence the proposed second, always-on, test.

> Alternatively, can we maybe use jq for this?

sure, `jq` would be nicer in this specific case, and I'm probably overthinking it, but I think it'd be nice to not rely on too many external tools and python is more widely useful.

But I'm totally ok with just slapping a `feature` statement on the whole file and calling it a day

Reply:
I'm fine with your proposed code, I'll amend the PR